src/app/aside/aside.component.scss

```diff
@@ -1,3 +1,11 @@
 aside{
-    background: chocolate;
+    background: honeydew;
+    height: 400px;
+}
+
+.active{
+    background: rgb(122, 230, 188);
```

Comment:
@Yuriy-27 please convert rgb to hex and don't rgb if there can be used hex (some minor performance advantage)

but if you need rgba (rgb with opacity) you can use rgba becahse hex doesn't support opacity.